### PR TITLE
add support for actualsub

### PR DESCRIPTION
**Pull request by omsmith**
Fixes https://github.com/Brightspace/node-auth-token/issues/10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,6 +64,14 @@ context.
 The tenant UUID this token belongs to. Not present outside of user and tenant
 contexts.
 
+---
+
+#### `.actualUser` -> `String|Undefined`
+
+The identifier for the acting user. For convenience, this will always be the
```
Comment (j3parker) [inline README.md]: Is it convenient? I guess so.

Comment (omsmith) [inline README.md]: You said so :P

**Comment by neverendingqs** [comment]
For tokens in an inconsistent state that have `actualsub`  but not `sub`, we're okay with treating them as service tokens?

**Comment by omsmith** [comment]
Sounds like a bit of a bug elsewhere at that point, not sure what I would do other than reject it, despite being told it's valid.

**Pull request merged**